README.md

```diff
@@ -35,6 +35,7 @@ import 'cypress-file-upload';
 ```
 
 Note: With Typescript, ensure the following is in your `cypress\tsconfig.json` file:
+
```

Comment:
My linter must have gotten a little eager, happy to revert the formatting changes if needed